src/main/scala/uk/gov/nationalarchives/consignmentexport/BagMetadata.scala

```diff
@@ -18,7 +18,14 @@ class BagMetadata(keycloakClient: KeycloakClient)(implicit val logger: SelfAware
     private def isStringNullOrEmpty(s: String): Boolean = s == null || s.trim.isEmpty
 
     def isUserRepresentationComplete: Boolean = {
-      !isStringNullOrEmpty(value.getFirstName) && !isStringNullOrEmpty(value.getLastName)
+      !isStringNullOrEmpty(value.getFirstName) && !isStringNullOrEmpty(value.getLastName) && !isStringNullOrEmpty(value.getEmail)
+    }
+
+    def getUserDetails(): UserDetails = {
```

Comment:
IntelliJ says that this could be simplified to a simple if-else. Do you think maybe it might be better as one?